packages/cli/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "webpack-cli",
```

Comment:
```suggestion
  "name": "@webpack-cli/core",
```

Reply:
We can do it later with another PR. 
Anyway, we need at least a package called `webpack-cli` because that's what the users will download.